shared/middleware/helpers/property_bag.rb

```diff
@@ -56,3 +56,93 @@ def to_hash()
   end
 
 end
+
+class DynamoPropertyBag
+
+  class NotFound < Sinatra::NotFound
+  end
+
+  def initialize(channel_id, storage_id)
+    channel_owner, @channel_id = storage_decrypt_channel_id(channel_id)
+    @storage_id = storage_id
+
+    @hash = "#{@channel_id}:#{storage_id}"
+  end
+
+  def db
+    @@dynamo_db ||= Aws::DynamoDB::Client.new(
+      region: 'us-east-1',
+      access_key_id: CDO.s3_access_key_id, 
+      secret_access_key: CDO.s3_secret_access_key, 
+    )
+  end
+
+  def delete(name)
+    begin
+      db.delete_item(
+        table_name:CDO.dynamo_properties_table,
+        key:{'hash'=>@hash,name:name},
```

Comment:
Space missing after colon.<br>Space missing after comma.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.